src/.eslintrc.json

```diff
@@ -1,17 +1,17 @@
 {
   "root": true,
-  "extends": "eslint-config-es5",
-  "parserOptions": {"ecmaVersion": 5},
-  "plugins": ["googleappsscript"],
+  "extends": ["eslint-config-es5", "prettier"],
+  "parserOptions": {"ecmaVersion": 6},
```

Comment:
The code in `src` directory will be uploaded to GAS editor w/o transpiling, this is the reason that we have two linters with different rules - one for `src` directory and the other for everything else(runs locally on node.js and won't be uploaded to GAS editor) 

You should keep the ecma version. 

More info on GAS JS version: 
https://developers.google.com/apps-script/guides/services/#basic_javascript_features
https://en.wikipedia.org/wiki/Google_Apps_Script